From 7f33b05ffab10af23032ceaf15a94c7709a3439f Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:09 -0400
Subject: [fs] btrfs: fix comments and printk msgs

Message-id: <1407179109-13714-3-git-send-email-sandeen@redhat.com>
Patchwork-id: 86960
O-Subject: [RHEL7.1 PATCH 002/422] btrfs: fix comments and printk msgs
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Masanari Iida <standby24x7@gmail.com>

commit 8faaaead62c3c7394fa6302303ce70e484b509ba
Author: Masanari Iida <standby24x7@gmail.com>
Date:   Tue Jan 7 21:58:06 2014 +0900

    treewide: fix comments and printk msgs

    This patch fixed several typo in printk from various
    part of kernel source.

    Signed-off-by: Masanari Iida <standby24x7@gmail.com>
    Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/tests/free-space-tests.c b/fs/btrfs/tests/free-space-tests.c
index 790f118..c8d9ddf 100644
--- a/fs/btrfs/tests/free-space-tests.c
+++ b/fs/btrfs/tests/free-space-tests.c
@@ -266,7 +266,7 @@ static int test_bitmaps_and_extents(struct btrfs_block_group_cache *cache)
  }
 
  if (test_check_exists(cache, 512 * 1024, 3 * 1024 * 1024)) {
-  test_msg("Left over peices after removing overlapping\n");
+  test_msg("Left over pieces after removing overlapping\n");
   return -1;
  }
 
-- 
1.7.1